docker/centrifuge-chain/Dockerfile

```diff
@@ -27,43 +27,56 @@ FROM --platform=linux/amd64 docker.io/paritytech/ci-linux:production as builder
+		org.opencontainers.image.title="centrifugeio/centrifuge-chain" \
+		org.opencontainers.image.description="Centrifuge, the layer 1 of RWA. This is the official Centrifuge (para)chain image" \
+		org.opencontainers.image.source="https://github.com/centrifuge/centrifuge-chain/" \
+		org.opencontainers.image.created="${BUILD_DATE}" \		
+		org.opencontainers.image.documentation="https://github.com/centrifuge/centrifuge-chain/blob/main/README.md" \
+		org.opencontainers.image.version="$VERSION" \
+		org.opencontainers.image.licenses="GPL-3" \
+		org.opencontainers.image.base.name="ubuntu/jammy" \
+		org.opencontainers.image.base.digest="ubuntu@sha256:6042500cf4b44023ea1894effe7890666b0c5c7871ed83a97c36c76ae560bb9b" 
+		# org.opencontainers.image.url= \		
 
-	LABEL io.centrifuge.image.authors="protocol@k-f.co" \
-		io.centrifuge.image.vendor="Centrifuge" \
-		io.centrifuge.image.title="centrifugeio/centrifuge-chain" \
-		io.centrifuge.image.description="Centrifuge, the layer 1 of RWA. This is the official Centrifuge (para)chain image" \
-		io.centrifuge.image.source="https://github.com/centrifuge/centrifuge-chain/blob/main/docker/centrifuge-chain/Dockerfile" \
-		io.centrifuge.image.created="${BUILD_DATE}"
 
-	# Add chain resources to image
-	COPY node/res /resources/
-
-	COPY --from=builder /centrifuge-chain/target/release/centrifuge-chain /usr/local/bin
 
-	RUN apt-get autoremove -y && apt-get clean && rm -rf /var/lib/apt/lists/* ;
+	RUN apt-get update && apt-get install -y curl jq
+	RUN apt-get autoremove -y && apt-get clean && rm -rf /var/lib/apt/lists/*
 
 	# Because of the following, all data and config directories need to be owned by UID and GID 1000
 	RUN useradd -m -u 1000 -U -s /bin/sh -d /centrifuge centrifuge
+	
+	COPY node/res /resources/
+	COPY --from=builder /centrifuge-chain/target/release/centrifuge-chain /usr/local/bin/
 
-	RUN mkdir -p /data && \
+	COPY docker/scripts /centrifuge/scripts
+	RUN chmod +x /centrifuge/scripts/* && \
+		ln -s /centrifuge/scripts/* /usr/local/bin/
+
+	RUN mkdir -p /data && \ 
 		chown -R centrifuge:centrifuge /data && \
 		chown -R centrifuge:centrifuge /resources && \
-		chown -R centrifuge:centrifuge /usr/local/bin/centrifuge-chain && \
-		chown -R centrifuge:centrifuge /centrifuge/
-
-	COPY ./docker/scripts/entrypoint.sh /centrifuge/entrypoint.sh
-	RUN chown -R centrifuge:centrifuge /centrifuge/entrypoint.sh && chmod +x /centrifuge/entrypoint.sh
+		chown -R centrifuge:centrifuge /centrifuge/ 
 
 # Running as an non-root is a good security practice
 # in some cases the container can be forced to run as root overriding the next line
 # but by default we want to enforce this.
 USER centrifuge
+WORKDIR /centrifuge
 
 # checks
 RUN ldd /usr/local/bin/centrifuge-chain && \
 	/usr/local/bin/centrifuge-chain --version
+RUN set -e && command -v check-node-ready && command -v entrypoint.sh
 
 EXPOSE 30333 9933 9944
 VOLUME ["/data"]
-ENTRYPOINT ["/centrifuge/entrypoint.sh"]
+ENTRYPOINT ["entrypoint.sh"]
```

Comment:
AFAICS, both scripts are not in the workdir `centrifuge` but `centrifuge/scripts`. Am I missing something here or are we not using the correct paths for the `check-node-ready` and `entrypoint.sh` scripts?

Reply:
RUN chmod +x /centrifuge/scripts/* && \
		ln -s /centrifuge/scripts/* /usr/local/bin/
This will make the scripts executable and also add them to PATH so they can be executed from anywhere